MAUI/DataGrid/loadMore.md

```diff
@@ -0,0 +1,195 @@
+
+To enable and load items at runtime, follow the code example:
+
+```C# 
+//Enable load more in SfDataGrid
+dataGrid.AllowLoadMore = true;
+dataGrid.LoadMoreCommand = new Command(ExecuteLoadMoreCommand);
+ 
+private async void ExecuteLoadMoreCommand()
+{
+    this.dataGrid.IsBusy = true;
+    await Task.Delay(new TimeSpan(0, 0, 5));
+    viewModel.LoadMoreItems();
+    this.dataGrid.IsBusy = false;
+} 
+
+//ViewModel.cs
+public OrderInfoRepository()
+{
+    orderInfo = this.GenerateOrders(25);
+}
+
+
+//ItemsSource
+private ObservableCollection<OrderInfo> orderInfo;
+public ObservableCollection<OrderInfo> OrderInfoCollection
+{
+    get { return orderInfo; }
+    set { this.orderInfo = value; }
+}
+
+//ItemsSource Generator
+public ObservableCollection<OrderInfo> GenerateOrders(int count)
+{
+    ObservableCollection<OrderInfo> orders = new ObservableCollection<OrderInfo>();
+    for (int i = 1000; i <= 1000 + count; i++)
+    {
+        var order = new OrderInfo(i, 
+            CustomerID[random.Next(7)],
+            Country[random.Next(7)], 
+            Customer[random.Next(7)], 
+            ShipCity[random.Next(7)]);
+        orders.Add(order);
+    }
+    return orders;            
+}
+
+public void LoadMoreItems()
+{
+    for (int i = 0; i < 10; i++)
+        this.OrderInfoCollection.Add(this.GenerateOrderInfo(OrderInfoCollection.Count));
+}
+
+
+public OrderInfo GenerateOrderInfo(int count)
+{
+    var order = new OrderInfo(
+        count + 1000, CustomerID[random.Next(7)],
+        Country[random.Next(7)],
+        Customer[random.Next(7)],
+        ShipCity[random.Next(7)]);
+
+    return order;
+}
+```
+![DataGridLoadMoreView](Images\loadmore\maui-datagrid-loadmoreview.gif)
+
+## Customize DataGridLoadMoreView
+
+### Customize DataLoadMoreView display text
+Customize the text displayed in the `LoadMoreView` by setting the `SfDataGrid.LoadMoreText` property as follows:
+
+```C#
+//setting load more text in SfDataGrid
+dataGrid.LoadMoreText = "Load More Items";
+//We need to set the width of the button according to the LoadMoreText
+dataGrid.LoadMoreView.Children.OfType<Button>().ToList()[0].WidthRequest = 150;
+```
+![DataGridLoadMore with LoadMoreText](Images\loadmore\maui-datagrid-loadmoreview-loadmoretext.png)
+
+### Customize DataLoadMoreView position
+Customize the displayed DataLoadMoreView to either top or bottom based on the requirement.
+
+```C#
+//Enable load more in SfDataGrid
+dataGrid.LoadMorePosition = DataGridLoadMorePosition.Top;
+```
+**Top**
+
+![DataGridLoadMore with LoadMorePosition Top](Images\loadmore\maui-datagrid-loadmoreview-loadmoreposition-top.png)
+
+
+**Bottom**
+
+![DataGridLoadMore with LoadMorePosition Bottom](Images\loadmore\maui-datagrid-loadmoreview-loadmoreposition-bottom.png)
+
+### Customize DataLoadMoreView Appearance
+We can customize the appearance of the DataGridLoadMoreView using the following properties.
```

Comment:
The appearance of the built-in load more view can be personalized through the following properties:

Reply:
Replaced the content